samples/nrf9160/secure_services/prj.conf

```diff
@@ -4,3 +4,4 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 CONFIG_REBOOT=y
+CONFIG_BOOTLOADER_MCUBOOT=y
```

Comment:
That should not be needed; this is a rather generic sample, that should run without MCU boot necessarily, no?

Reply:
It is there to have a valid range to test. Otherwise no area would be readable through the secure services.

Reply:
Consider having ifdefs around the MCUBOOT-stuff, so it will work with MCUBOOT disabled

Reply:
I mean, this is a generic sample demonstrating secure services - only needs an SPM to work; why having mcu-boot for that? Why not making a new sample, or at least, a new project configuration file?

Reply:
It will compile without mcuboot, but there is no valid data ranges to read, and I don't want to change the subsys code only to make the sample work better.

Due to the magic TZ macros I don't think this can be unit tested trivially, but maybe I'm wrong.

Reply:
This is a sample demonstrating the secure services; it only needs an SPM (in theory, not even that; it needs some sort of secure firmware with the same services). I want to be able to run it on nRF9160 without MCUboot.

If you really need MCUboot for the sample to work, can you add an alternative prj.conf file?

Reply:
I will do as you say. However, I fear that we will get code rot unless this is being checked under CI, so I will task to the test engineers about this :)